src/mainAppBar/userProfile.component.test.tsx

```diff
@@ -95,6 +95,25 @@ describe('User profile component', () => {
     ).toBeTruthy();
   });
 
+  it('opens cookie policy/management page if manage cookies clicked', () => {
+    const testStore = mockStore(state);
+    const wrapper = mount(
+      <Provider store={testStore}>
+        <UserProfileComponent />
+      </Provider>
+    );
+
+    // Click the user menu button and click on the manage cookies menu item.
+    wrapper.find('button').simulate('click');
+    wrapper
+      .find(MenuItem)
+      .first()
```

Comment:
Since you've added aria-labels to these - you can select by them instead. Means if another menu item comes along these tests will still pass

Reply:
@louise-davies I have left selecting by the component in as the wrapper has it under a forward-ref meaning it cannot be selected easily.

Reply:
@louise-davies I have changed to using IDs, as you mentioned, in https://github.com/ral-facilities/scigateway/pull/133/commits/2ef5e5cee6b3d6c4ba82f54b4e85fb1c8f29404b.